### DISCUSSION TITLE
Salário menor para trabalho remoto

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#586] [Discussão]
Recentemente recebi uma proposta de trabalho e já havia nele o range do valor da contratação. Quando comentei que só estava aberto a oportunidades que fosse remoto recebi a resposta: "... 3 dias no escritório e 2 dias home office. Salário a combinar devido a condição de home office". 
Não dei sequência pois não me surgiu o interesse, então não sei o quanto eles estariam disposto a pagar. 

Vejo pouquíssimas vagas oferecendo home office, elas são realmente mais desvalorizadas? Como vocês avaliam esse pensamento de salário menor por ser home office?

**Comment by wmsbill** [comment]
Cara, Salário para mim tem que ser atribuído ao valor que você gera para a empresa, não importando a modalidade de trabalho. Dito isto, se eles alegarem que vc é um sênior e eles tem um time mid level que eles querem fazer evoluir, seria uma desculpa plausível para querer te pagar menos. Fora isso, não tenho a mínima idéia.

**Comment by rubenmarcus** [comment]
Po, vejo vagas remotas em outros países, pagando muito viu, quando voce converte por exemplo 2000 dolares para real = 6593 reais, Acho que é mais cultura mesmo. 
Vejo aqui no Brasil, duas coisas: 
-Empresas que tratam funcionários de TI, apenas como uma engrenagem que eles podem trocar e adequar as necessidades que eles quiserem, e vejo também um mercado subvalorizado, tendo em vista que tem muitos profissionais bons aqui,em comparação a muitos outros países.


Mas isso é meio que global, vagas remotas e homeoffice são minoria ainda, então vai do que o @wmsbill  falou mesmo

**Comment by rubenmarcus** [comment]
PS: eu também estou a procura de trampo remoto. Me dou melhor trabalhando home-office, e pra mim seria perfeito um estilo de vida nomade. Eu to na busca também, to só preparando um material esses meses, e estudando, pra ver se arrumo vagas em qualquer lugar do mundo que seja, importante é que seja remoto.

**Comment by felipefialho** [comment]
@felipechama Nunca percebi diferença nos salários. E nem deveria.

**Comment by rafael1995** [comment]
@felipechama trabalhei durante um ano sendo totalmente Home-office, a empresa até hoje tem preferencia nesse modo de trabalho, até mesmo por ser interessante economicamente, sem gastos com ambiente e selecionando pessoas de regiões que normalmente a mão de obra é mais barata.
No meu ponto de vista, não faz sentido um ajuste salarial por este motivo.. sendo possível o funcionário realizar todas as funções normalmente.

**Comment by ghost** [comment]
Eu compartilho 100% da visão de vocês. Achei muito estranho a atitude da empresa de rever o valor salarial após o pedido de home office. A primeira coisa que imaginei, foi que eu trabalharia normal a jornada de 8h e o valor do salário se baseia em conhecimento técnico. 
Como o @wmsbill comentou, seria uma desculpa para querer pagar menos.

**Comment by bernardodiasc** [comment]
Qq coisa voce pede pra rever o valor pra mais, afinal, os custos de ter um profissional _in loco_ são bem mais altos. E esses custos, por sinal, recaem sob o profissional remoto. Pra citar alguns, luz, equipamentos, espaço, transporte, alimentação, etc...

**Comment by remacedo** [comment]
Muitas empresas ainda não possuem a cultura de Home Office, não fazem um bom controle das atividades através de um gerenciamento seguro e rotulam o trabalho à distância como menos produtivo, consequentemente reduzem o valor acreditando em um menor potencial de entrega. Empresas que possuem esta cultura, certamente manterão o mesmo patamar de valores.

**Comment by felipeuntill** [comment]
Só lembrando alguns amigos, muita gente entende que o fato de receber internacionalmente ou informalmente os isenta de qualquer contribuição para o Temer, porém na verdade  não é bem assim.

Nosso regime tributário nos [obriga a declarar qualquer renda recorrente](https://economia.uol.com.br/imposto-de-renda/noticias/redacao/2017/02/07/voce-pode-ja-estar-devendo-para-a-receita-antes-mesmo-de-declarar-seu-ir.htm) e se você for pego e acredite, pode demorar mais você será...

_Tudo que passa por um banco, passa pelo banco central e se passa pelo banco central o temer está de olho!_

**Comment by rubenmarcus** [comment]
@felipeuntill  http://www.wikihow.com/Open-a-Bahamas-Bank-Account

ahahahahahahha, to brincando!

**Comment by rubenmarcus** [comment]
acredito que alguem que receba em bitcoin, não precise pagar o governo

**Comment by besopmac** [comment]
Existem várias formas lícitas de você manter seu dinheiro longe do Brasil. Sugiro que dêem uma olhada no site [http://www.investidorinternacional.com/](url). E, sim, o Bitcoin é uma forma de reduzir a dentada da boquinha gulosa do nosso glorioso pior sócio do munod: o Estado.
